src/BLSApkRegistry.sol

```diff
@@ -123,18 +123,27 @@ contract BLSApkRegistry is BLSApkRegistryStorage {
             params.pubkeyG1.Y, 
             params.pubkeyG2.X, 
             params.pubkeyG2.Y, 
-            pubkeyRegistrationMessageHash.X, 
-            pubkeyRegistrationMessageHash.Y
+            _pubkeyRegistrationMessageHash.X, 
+            _pubkeyRegistrationMessageHash.Y
         ))) % BN254.FR_MODULUS;
         
         // e(sigma + P * gamma, [-1]_2) = e(H(m) + [1]_1 * gamma, P') 
         require(BN254.pairing(
             params.pubkeyRegistrationSignature.plus(params.pubkeyG1.scalar_mul(gamma)),
             BN254.negGeneratorG2(),
-            pubkeyRegistrationMessageHash.plus(BN254.generatorG1().scalar_mul(gamma)),
+            _pubkeyRegistrationMessageHash.plus(BN254.generatorG1().scalar_mul(gamma)),
             params.pubkeyG2
         ), "BLSApkRegistry.registerBLSPublicKey: either the G1 signature is wrong, or G1 and G2 private key do not match");
 
+        //if keys already registered checkpoint the previous pubkey and pubkeyHash of the operator
```

Comment:
I found this challenging to parse. I feel like there's an implied comma between `if keys already registered` and `checkpoint the previous pubkey and pubkeyHash of the operator`? I'd recommend just inserting the word `then` in between these two phrases to make it clearer